style.css

```diff
@@ -0,0 +1,129 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  background-color: #f7f7f9;
+}
+
+html {
+  width: 375px;
+}
+
+li {
+  list-style: none;
+  padding-top: 10px;
+  padding-right: 12px;
+}
+
+.color {
+  color: #6070ff;
+}
+
+.pos {
+  position: relative;
+}
+
+.flex {
+  display: flex;
+}
+
+button {
+  background-color: transparent;
+  border: none;
+  outline: none;
+}
+
+/* HEADER  */
+.header {
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px;
+  padding: 12px;
+  text-align: center;
+}
+
+.header a {
+  font-size: 24px;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+.logo:hover {
+  font-weight: 900;
+  font-size: 30px;
+  color: chocolate;
+}
+
+button:hover {
+  font-size: 30px;
+}
+
+/* INTRODUCTION */
+#ell2 {
+  left: 327px;
+}
+
+#ell1 {
+  right: -72px;
+  top: -12px;
+  background-color: transparent;
+}
+
+#rect1 {
+  right: -77px;
+  top: -28px;
+  background-color: transparent;
+}
+
+#ell3 {
+  background-color: transparent;
+  left: -32px;
+}
+
+.intro-text {
+  flex-direction: column;
+  padding: 12px 30px;
+}
```

Comment:
Kindly use a `background-image`  for decorations in this section rather than images. This is a requirement and is best practices. You can download the image by using this tutorial

![how_to](https://user-images.githubusercontent.com/12239821/176725714-a1998aa6-7258-40b1-92ea-96a7be791abf.png)